src/rocksdb/src/tuning.rs

```diff
@@ -276,15 +286,28 @@ impl FromStr for CompressionType {
     type Err = anyhow::Error;
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
-        match s {
-            "zstd" => Ok(Self::Zstd),
-            "snappy" => Ok(Self::Snappy),
-            "lz4" => Ok(Self::Lz4),
-            "None" => Ok(Self::None),
-            other => Err(anyhow::anyhow!(
-                "{} is not a supported compression type",
-                other
-            )),
+        let s = UncasedStr::new(s).as_str();
+        if s == "zstd" {
+            Ok(Self::Zstd)
+        } else if s == "snappy" {
+            Ok(Self::Snappy)
+        } else if s == "lz4" {
+            Ok(Self::Lz4)
+        } else if s == "None" {
```

Comment:
this should be `"none"` for consistency, as its distinct from Option::None

Reply:
ah, was just copying what was already here; will fix.